pkg/cmd/util/spinner/spinner_group.go

```diff
@@ -0,0 +1,167 @@
+package multispinner
```

Comment:
Can we just call multispinner spinnerlib?

Reply:
Is it ok to change `multispinner` to `terminalspinner`?

Reply:
or just spinners

Reply:
If I use `spinner*` as the package name, the IDE has this message: "Name starts with the package name", does it matter?

Reply:
What if changing `spinner/*.go  to spinners/*.go` and then using spinners as pkg name?